securedrop_client/gui/widgets.py

```diff
@@ -1211,9 +1261,9 @@ def on_authentication_changed(self, authenticated: bool):
         event instead of 'toggled' event when not authenticated because toggling will be disabled.
         """
         if authenticated:
-            self.toggled.connect(self.on_toggle)
+            self.enable()
         else:
-            self.pressed.connect(self.on_toggle_offline)
```

Comment:
did adding a `self.toggled.disconnect()` and `self.pressed.disconnect()` before this line not work?

Reply:
It did.